.babelrc

```diff
@@ -1,4 +1,4 @@
 {
   "presets": ["es2015", "react"],
-  "plugins": ["transform-object-assign", "transform-object-rest-spread"],
+  "plugins": ["transform-object-assign", "transform-object-rest-spread", "transform-async-to-generator"],
```

Comment:
this will need to be added to terra-ui too
https://github.com/cerner/terra-ui/blob/master/.babelrc

Reply:
Is this noted in the docs to add this plugin to use the form validation?

Reply:
Yup. It's added here (https://github.com/cerner/terra-framework/pull/165/commits/75ea4deb024354c8a894f7a7ecbb98c406ab931a#diff-e4bbdd7f11ee47faf0887a548d37aa18R60). Also, it's not needed for form-validation unless you have the need for integrating promises for ajax requests using async and await